apps/jobs/utils/validators.py

```diff
@@ -18,31 +19,45 @@ class validationClass:
     2. Image files
     """
 
+    logger = logging.getLogger("jobs")
+
     @staticmethod
     def is_valid_uuid(value):
         # Expects value in proper format(with hyphens) of uuid, returns bool value
+        validationClass.logger.info("Getting valid uuid with proper format")
         try:
             uuid_value = uuid.UUID(str(value))
         except ValueError:
+            validationClass.logger.error("The specified value is not a valid uuid")
             return False
         else:
-            return str(uuid_value) == str(value)
+            if str(uuid_value) == str(value):
+                return True
+            else:
+                # Log an error when the UUID value is not the same as the input
+                validationClass.logger.error(
+                    f"Mismatch between input and UUID value: {value}"
+                )
+                return False
 
     @staticmethod
     def validate_id(uuid, idtype: str, model_class):
         """perform checks on uuid, and if it
         exists in the database."""
-
+        validationClass.logger.info("validating the id type")
         if not validationClass.is_valid_uuid(uuid):
+            validationClass.logger.error(f"{idtype} isn't a valid UUID")
             return {"error": f"{idtype} isn't a valid UUID"}
 
         if model_class.objects.filter(pk=uuid).count() < 1:
+            validationClass.logger.error(f"This {idtype} doesn't exist")
             return {"error": f"This {idtype} doesn't exist"}
 
     def image_validation(self, image_file):
         # check size
         filesize = image_file.size / (1024 * 1024)
         if filesize > 10:
+            validationClass.logger.warning(f"{image_file} exceeds {filesize}")
```

Comment:
Don't print the `{image_file}` here, suppose, from the attacker's perspective if an attacker uploads an arbitrary file with N number of 0s padded to its name or if the image file name is too long, then it'd be difficult for us to read the logs.